### ISSUE TITLE
What function does Availability true/false perform? Can estimated date be displayed?

### ISSUE THREAD

**Issue by janetc** [PTFS-Europe/koha#76]
Feedback from MHRA:
The availability says true/false but at this point there is no actual date – apparently on the BLDSS site it  clearly shows an estimated despatch date on which you can base your decision to place the request. Could this functionality be available with the api?

**Comment by a-sassmannshausen** [comment]
Right.  This would be low-hanging fruit, I reckon, now that we have authenticated availability requests.
